test/constants_test.dart

```diff
@@ -57,8 +57,7 @@ void main() {
           const Super() : super(aString: 'Super Hello');
         }
       ''');
-      constants = resolver
-          .getLibraryByName('test_lib')
+      constants = (await resolver.findLibraryByName('test_lib'))
```

Comment:
probably - I missed some of these if they didn't cause the tests to fail. Perhaps another argument for a breaking change in the signature in build_test.